controllers/admin/AdminTranslationsController.php

```diff
@@ -2191,7 +2191,32 @@ public function getListModules($withInstance = false)
 
         return $modules;
     }
+    
+    /**
+     * @param $directory : name of directory
+     * @return array
+     */
+    protected function getSubjectMailContent($directory)
+    {
+        $subject_mail_content = array();
+
+        if (Tools::file_exists_cache($directory.'/lang.php')) {
+            // we need to include this even if already included (no include once)
+            include($directory.'/lang.php');
+            foreach ($GLOBALS[$this->translations_informations[$this->type_selected]['var']] as $key => $subject) {
+                $this->total_expression++;
```

Comment:
```suggestion
                ++$this->total_expression;
```

Reply:
HI,
Thanks for the suggestion but i don't think that there is any need of pre-increment of $this->total_expression variable

Reply:
HI,
I understand. It is mandatory for PHPCS Fixer